src/extensionsIntegrated/Phoenix-live-preview/main.js

```diff
@@ -389,6 +411,7 @@ define(function (require, exports, module) {
         $iframe = $panel.find("#panel-live-preview-frame");
         $pinUrlBtn = $panel.find("#pinURLButton");
         $highlightBtn = $panel.find("#highlightLPButton");
+        $wordNavigationBtn = $panel.find("#wordNavigationLPButton");
```

Comment:
We don't need a new button for this, we can remove all these code as it can be tied to `toggleLiveHighlight` button.